diff --git a/models/mlp_hopper/mlp_hopper_0.zip b/models/mlp_hopper/mlp_hopper_0.zip
index 825dcb2..bee77c2 100644
Binary files a/models/mlp_hopper/mlp_hopper_0.zip and b/models/mlp_hopper/mlp_hopper_0.zip differ
diff --git a/models/mlp_hopper/mlp_hopper_1.zip b/models/mlp_hopper/mlp_hopper_1.zip
index d762d66..38324a6 100644
Binary files a/models/mlp_hopper/mlp_hopper_1.zip and b/models/mlp_hopper/mlp_hopper_1.zip differ
diff --git a/models/mlp_hopper/mlp_hopper_2.zip b/models/mlp_hopper/mlp_hopper_2.zip
index 7804401..415efeb 100644
Binary files a/models/mlp_hopper/mlp_hopper_2.zip and b/models/mlp_hopper/mlp_hopper_2.zip differ
diff --git a/models/mlp_hopper/mlp_hopper_3.zip b/models/mlp_hopper/mlp_hopper_3.zip
index f117f0f..3a51304 100644
Binary files a/models/mlp_hopper/mlp_hopper_3.zip and b/models/mlp_hopper/mlp_hopper_3.zip differ
diff --git a/models/mlp_hopper/mlp_hopper_4.zip b/models/mlp_hopper/mlp_hopper_4.zip
index 46d87eb..6e9fb68 100644
Binary files a/models/mlp_hopper/mlp_hopper_4.zip and b/models/mlp_hopper/mlp_hopper_4.zip differ
diff --git a/models/mlp_hopper/mlp_hopper_5.zip b/models/mlp_hopper/mlp_hopper_5.zip
index 08d2ba5..b405c7b 100644
Binary files a/models/mlp_hopper/mlp_hopper_5.zip and b/models/mlp_hopper/mlp_hopper_5.zip differ
diff --git a/models/mlp_hopper/mlp_hopper_6.zip b/models/mlp_hopper/mlp_hopper_6.zip
index 8777bf5..d1a7dd8 100644
Binary files a/models/mlp_hopper/mlp_hopper_6.zip and b/models/mlp_hopper/mlp_hopper_6.zip differ
diff --git a/models/mlp_hopper/mlp_hopper_7.zip b/models/mlp_hopper/mlp_hopper_7.zip
index 63f568e..cad1f85 100644
Binary files a/models/mlp_hopper/mlp_hopper_7.zip and b/models/mlp_hopper/mlp_hopper_7.zip differ
diff --git a/models/mlp_hopper/mlp_hopper_8.zip b/models/mlp_hopper/mlp_hopper_8.zip
index dfe7703..4123adf 100644
Binary files a/models/mlp_hopper/mlp_hopper_8.zip and b/models/mlp_hopper/mlp_hopper_8.zip differ
diff --git a/wandb/debug-internal.log b/wandb/debug-internal.log
index 66ed6d5..249a1a0 120000
--- a/wandb/debug-internal.log
+++ b/wandb/debug-internal.log
@@ -1 +1 @@
-run-20260610_090142-elfmh1yz/logs/debug-internal.log
\ No newline at end of file
+run-20260610_133331-8g34gq6f/logs/debug-internal.log
\ No newline at end of file
diff --git a/wandb/debug.log b/wandb/debug.log
index b33e8a0..ce1972b 120000
--- a/wandb/debug.log
+++ b/wandb/debug.log
@@ -1 +1 @@
-run-20260610_090142-elfmh1yz/logs/debug.log
\ No newline at end of file
+run-20260610_133331-8g34gq6f/logs/debug.log
\ No newline at end of file
diff --git a/wandb/latest-run b/wandb/latest-run
index 56b5802..ebc0914 120000
--- a/wandb/latest-run
+++ b/wandb/latest-run
@@ -1 +1 @@
-run-20260610_090142-elfmh1yz
\ No newline at end of file
+run-20260610_133331-8g34gq6f
\ No newline at end of file
